compose/test_keyspace_vschema.json

```diff
@@ -31,7 +31,7 @@
 			"type": "hash"
 		},
 		"messages_message_lookup": {
-			"type": "lookup_hash",
+			"type": "lookup_hash_unique",
```

Comment:
ApplySchema accepts either `lookup_hash` or `lookup_hash_unique.`
In this sense, it doesn't have to change to `lookup_hash_unique.`

If you try CreateLookupVindex, you will encounter the error like below.
CreateLookupVindex seems stricter, so I followed this error message for now.

> CreateLookupVindex Error: rpc error: code = Unknown desc = non-unique vindex 'from' should have more than one column: type:"lookup_hash" params:{key:"autocommit" value:"true"} params:{key:"from" value:"message"} params:{key:"table" value:"lookup_keyspace.messages_message_lookup_fix"} params:{key:"to" value:"page"} owner:"messages"